modules/gltf/doc_classes/GLTFSkeleton.xml

```diff
@@ -21,6 +21,7 @@
 		<method name="get_godot_bone_node">
 			<return type="Dictionary" />
 			<description>
+				Returns a [Dictionary] that maps skeleton bone indices to the indices of the GLTF nodes that the bones came from. In a GLTF file, a bone is a node, and Godot converts it to a skeleton bone.
```

Comment:
I think I get it.

This does a three level mapping:

1. gltf_skeleton is ignored
2. when we generate a godot_skeleleton a gltf_skeleton is generated
3. the gltf_skeleton maps a godot engine skeleton3d node to a bone to a gltf_node